spyder/plugins/workingdirectory.py

```diff
@@ -283,11 +283,20 @@ def chdir(self, directory, browsing_history=False,
             self.histindex = len(self.history)-1
         
         # Changing working directory
-        os.chdir(directory)
+        try:
+            FileNotFoundError
+        except NameError:
+            if os.name == 'nt':
+                FileNotFoundError = WindowsError
+            else:
+                FileNotFoundError = IOError
+        try:

```

Comment:
I think is better to catch `OSError` that's available in python2 and python3

Reply:
I agree with @rlaverde. @dalthviz, please simplify this code and only catch `OSError` here.